mybatis-spring-boot-test-autoconfigure/pom.xml

```diff
@@ -30,18 +30,37 @@
   <name>mybatis-spring-boot-test-autoconfigure</name>
   <dependencies>
     <dependency>
-      <groupId>org.mybatis.spring.boot</groupId>
-      <artifactId>mybatis-spring-boot-starter</artifactId>
+      <groupId>org.springframework.boot</groupId>
```

Comment:
I think `optional=true` is one choice. 
However I think `optional=true` is not appropriate in this case because a feature(`@MybatisTest`) that provided by `mybatis-spring-boot-test-autoconfigure` is need to these artifacts(`spring-boot-test-autoconfigure` and `spring-test`). If `mybatis-spring-boot-test-autoconfigure` has been provided other feature that does not depend on these artifact, i think `optional=true` is correct choice.
WDYT ?